docs/tpm.md

```diff
@@ -39,11 +39,11 @@ $ ./install_tpm_libs.sh -i
 # Command to uninstall TPM libraries
 $ ./install_tpm_libs.sh -u
 ```
-> **Note:** Installation of these components may require elevated permissions. Please use 'sudo' to execute the script.
+> ***Note:*** Installation of these components may require elevated permissions. Please use 'sudo' to execute the script.
 
 ### 2.1 Building and Installing Libraries for Trusted Platform Module (TPM)
 
-Following steps should be performed if SDO TPM [TPM-Library-Installation-Script](../utils/install_tpm_libs.sh) script is not used to setup SDO TPM libraries. Install only tpm2-tss library to enable TPM enabled SDO Client-SDK code compilation. To enable compilation and execution of TPM enabled SDO Client-SDK code install all three libraries namely: tpm2-tss, tpm2-abrmd, tpm2-tools and tpm2-tss-engine
+Following steps should be performed if SDO TPM [TPM-Library-Installation-Script](../utils/install_tpm_libs.sh) script is not used to setup SDO TPM libraries. Install only tpm2-tss library to enable TPM enabled SDO Client-SDK code compilation. To enable compilation and execution of TPM enabled SDO Client-SDK code install all three libraries namely: tpm2-tss, tpm2-abrmd, tpm2-tools, and tpm2-tss-engine
```

Comment:
Comma after 'TPM enabled SDO Client-SDK code' and full-stop after 'tpm2-tss-engine' in the last statement?

Reply:
Done.